<!DOCTYPE html>
<script src="./js/loadhtml.js"></script>

<html lang="en">

<head>
    <div w3-include-html="./html/header.html"></div>
    <script>
        w3IncludeHTML();
    </script>
</head>

<body>
    <div id="loading"></div>
    <div w3-include-html="./html/navleft.html"></div>
    <div w3-include-html="./html/navright.html"></div>
    <script>
        w3IncludeHTML();
    </script>

    <div class="container-fluid p-0">
        <section class="resume-section p-3 p-lg-5 d-flex d-column" id="about">
            <div class="my-auto">
                <div class="row">
                    <div class="brandcolumn">
                        <a class="navbar-brand js-scroll-trigger" href="#">
                            <span class="d-none d-lg-block">
                    <img class="img-fluid img-profile rounded-circle mx-auto mb-2" id="profileimg" src="./img/profiles/profile500.jpg" alt="">
                    </span></a>
                    </div>
                    <div class="contactcolumn">
                        <h1 class="mb-0" style="display:inline;text-transform:none;">Ba<span><span class="letter color-change">.</span><span class="stack">ı</span></span>han Lin</h1>
                        <!-- <h2 style="font-size:75px;display:inline;">aihan</h2>&nbsp;&nbsp; -->
                        <!-- <span class="text-primary"> -->
                        <!-- <h1 class="mb-0" style="display:inline"> L</h1><h2 class="mb-0" style="font-size:75px;display:inline">in</h2></span> -->
                        <div class="subheading mb-5" style="text-transform:none">Columbia University · New York City · (206) 915-1164 ·
                            <a href="mailto:doerlbh@gmail.com">doerlbh@<span style="display:none;">-ignoreme-</span>gmail.com</a>
                            <br/>
                        </div>
                    </div>
                    <p class="mb-5 text-primary">
                        I am a Ph.D. candidate pursuing Computational Neuroscience in the <a href="https://ctn.zuckermaninstitute.columbia.edu/"><u>Center for Theoretical Neuroscience</u></a> and <a href="https://zuckermaninstitute.columbia.edu/"><u>Zuckerman Mind Brain Behavior Institute</u></a> at Columbia University.
                        <!-- , with <a href="https://zuckermaninstitute.columbia.edu/nikolaus-kriegeskorte-phd"><u>Prof. Niko Kriegeskorte</u></a> and <a href="https://zuckermaninstitute.columbia.edu/ning-qian-phd"><u>Prof. Ning Qian</u></a>.  -->
                        I develop neuroscience-inspired AI systems and apply ML to understand human brains (Neuro &harr; AI).
                        <!-- I am interested in a variety of stuff ranging from geometric topology and dynamical systems to Bayesian statistics and machine learning, as well as their applications to neuroscience, genomics, and cognitive sciences. Most recently, I am developing neuroscience-inspired AI systems and applying ML to understand human brains (Neuro &harr; AI).  -->
                    </p>
                    <p class="mb-5 text-primary">
                        I graduated from the University of Washington, Seattle (UW), as part of the NIH-funded Computational Neuroscience Training Program, with a B.S. in Applied & Computational Mathematics and a B.A. in Psychology with Honors. I still maintain close collaborations and affliations with <a href="https://systemsbiology.columbia.edu/people/baihan-lin"><u>Columbia Department of Systems Biology</u></a> and <a href="https://amath.washington.edu/people/baihan-lin"><u>UW Department of Applied Mathematics</u></a>.
                    </p>
                    <p class="mb-5 text-primary">
                        I am into:
                        <br/>
                        <span class="field">
                        <img src="./img/fields/brain.png" alt="" style="width:80%" class="field-img" id="field-b"><br/>
                        brain
                    </span>
                        <span class="field">
                        <img src="./img/fields/robot.png" alt="" style="width:80%" class="field-img" id="field-r"><br/>
                        intelligence
                    </span>
                        <span class="field">
                        <img src="./img/fields/topology.png" alt="" style="width:80%" class="field-img" id="field-t"><br/>
                        topology
                    </span>
                        <span class="field">
                        <img src="./img/fields/psych.png" alt="" style="width:80%" class="field-img" id="field-p"><br/>
                        psychology
                    </span>
                        <span class="field">
                        <img src="./img/fields/design.png" alt="" style="width:80%" class="field-img" id="field-d"><br/>
                        design
                    </span>
                        <span class="field">
                        <img src="./img/fields/art.png" alt="" style="width:80%" class="field-img" id="field-a"><br/>
                        perception
                    </span>
                    </p>
                    <p class="handwritten" id="questions">
                        ?
                    </p>

                    <!-- 
                    <div class="subheading mb-3">Recent news</div>
                    <ul class="fa-ul mb-0">
                      <li>
                        <i class="fa-li fa fa-coffee"></i>
                        website still under construction</li>
                    </ul>
                    </div> -->
                </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="research">
            <div class="my-auto">
                <h2 class="mb-5">Research</h2>

                <p class="mb-5 text-primary">
                    I am very grateful to have mentors and collaborators with whom I have the honor to work on various interesting problems spanning <span style="color: blue;"> neuroscience (NS)</span>, <span style="color: red;">machine learning (ML)</span>, <span style="color: chocolate;"> computer vision (CV)</span>, <span style="color: purple;"> geometric/topological inference (GT)</span>, <span style="color: forestgreen;"> systems biology/genomics (SB)</span>, <span style="color: teal;"> applied mathematics (AM)</span>, <span style="color: brown;"> human-computer interaction (HCI)</span>, <span style="color: orange;"> Bayesian statistics (BS)</span>, and <span style="color: salmon;"> translational medicine/psychiatry (TM) </span>. I also maintain close industrial collaborations with IBM Research and Microsoft Research.
                </p>

                <p class="mb-5 text-primary">
                    <table align="left" class="mb-5 text-primary">
                        <tr bgcolor="lightgrey">
                            <th> From </th>
                            <th> To </th>
                            <th> At </th>
                            <th> With </th>
                            <th> Published </th>
                            <th> On </th>
                        </tr>
                        <tr>
                            <td> 2017 </td>
                            <td> now </td>
                            <td> IBM </td>
                            <td> CompNeuro & AI Foundations Groups</td>
                            <td> <a href="#p7">[7]</a><a href="#p8">[8]</a><a href="#p11">[11]</a><a href="#p15">[15]</a><a href="#p18">[18]</a><a href="#p19">[19]</a> </td>
                            <td> <span style="color: blue;">NS</span>,<span style="color: red;">ML</span>,<span style="color: chocolate;">CV</span>,<span style="color: purple;">GT</span>,<span style="color: salmon;">TM</span> </td>
                        </tr>
                        <tr>
                            <td> 2017 </td>
                            <td> 2017 </td>
                            <td> Microsoft </td>
                            <td> EPIC & HCI Groups </td>
                            <td> <a href="#p20">[20]</a></td>
                            <td> <span style="color: blue;">NS</span>,<span style="color: chocolate;">CV</span>,<span style="color: brown;">HCI</span></td>
                        </tr>
                        <tr>
                            <td> 2018 </td>
                            <td> now </td>
                            <td> Columbia </td>
                            <td> Visual Inference Lab (PI: Kriegeskorte) </td>
                            <td> <a href="#p9">[9]</a><a href="#p14">[14]</a><a href="#p17">[17]</a></td>
                            <td> <span style="color: blue;">NS</span>,<span style="color: red;">ML</span>,<span style="color: chocolate;">CV</span>,<span style="color: purple;">GT</span> </td>
                        </tr>
                        <tr>
                            <td> 2018 </td>
                            <td> now </td>
                            <td> </td>
                            <td> Neuro Theory Center (PI: Qian) </td>
                            <td> <a href="#p10">[10]</a><a href="#p21">[21]</a></td>
                            <td> <span style="color: blue;">NS</span>,<span style="color: red;">ML</span>,<span style="color: chocolate;">CV</span>,<span style="color: purple;">GT</span>,<span style="color: orange;">BS</span> </td>
                        </tr>
                        <tr>
                            <td> 2018 </td>
                            <td> 2018 </td>
                            <td> </td>
                            <td> Math Genomics Lab of DSB (PI: Rabadan) </td>
                            <td> <a href="#p13">[13]</a><a href="#p17">[17]</a></td>
                            <td> <span style="color: forestgreen;">SB</span>,<span style="color: red;">ML</span>,<span style="color: purple;">GT</span> </td>
                        </tr>
                        <tr>
                            <td> 2017 </td>
                            <td> 2017 </td>
                            <td> </td>
                            <td> Comp Bio Lab of CS (PI: Pe’er) </td>
                            <td> <a href="#p6">[6]</a>&nbsp;</td>
                            <td> <span style="color: forestgreen;">SB</span>,<span style="color: red;">ML</span>,<span style="color: salmon;">TM</span> </td>
                        </tr>
                        <tr>
                            <td> 2017 </td>
                            <td> 2017 </td>
                            <td> UW </td>
                            <td> UbiComp Lab of CSE & EE (PI: Patel) </td>
                            <td> </td>
                            <td> <span style="color: brown;">HCI</span>,<span style="color: red;">ML</span>,<span style="color: salmon;">TM</span> </td>
                        </tr>
                        <tr>
                            <td> 2015 </td>
                            <td> 2017 </td>
                            <td> </td>
                            <td> Institute for Protein Design (PI: Baker) </td>
                            <td> <a href="#p12">[12]</a></td>
                            <td> <span style="color: forestgreen;">SB</span> </td>
                        </tr>
                        <tr>
                            <td> 2016 </td>
                            <td> 2016 </td>
                            <td> </td>
                            <td> Biophysics Lab of Applied Math (PI: Qian) &nbsp; &nbsp;</td>
                            <td> </td>
                            <td> <span style="color: teal;">AM</span>,<span style="color: forestgreen;">SB</span></td>
                        </tr>
                        <tr>
                            <td> 2014 </td>
                            <td> 2016 </td>
                            <td> </td>
                            <td> Vision Neuroscience Lab (PI: Olavarria) </td>
                            <td> <a href="#p2">[2]<a><a href="#p16">[16]</a></td>
                            <td> <span style="color: blue;">NS</span>,<span style="color: chocolate;">CV</span> </td>
                        </tr>
                        <tr>
                            <td> 2014 </td>
                            <td> 2017 </td>
                            <td> BIME </td>
                            <td> State Key Lab of Pathogen & Biosecurity </td>
                            <td> <a href="#p1">[1]</a><a href="#p3">[3]<a><a href="#p4">[4]</a><a href="#p5">[5]</a></td>
                            <td> <span style="color: forestgreen;">SB</span>,<span style="color: teal;">AM</span>,<span style="color: salmon;">TM</span></td>
                        </tr>
                        <tr>
                            <td> 2013 </td>
                            <td> 2013 </td>
                            <td> BGI </td>
                            <td> Personalized Genome Group </td>
                            <td> </td>
                            <td> <span style="color: forestgreen;">SB</span>,<span style="color: salmon;">TM</span></td>
                        </tr>
                    </table>
                </p>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="publications">
            <div class="my-auto">
                <h2 class="mb-5">Publications</h2>
                <p class="mb-5 text-primary"> <b>21</b> total = <b>11</b> conferences + <b>6</b> journals + <b>4</b> manuscript / arXiv preprints &nbsp;
                    <br /> (click on <b> abstract </b> to learn more) &nbsp; </p>
                <h3> ongoing...</h3>
                <br />
                <ul class="fa-ul mb-0">
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2019unsupervised.png" alt="lin2019reinforce" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [21]&nbsp;<b class="text-primary" id="p21"> Unsupervised Attention Mechanism across Neural Network Layers &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>
                                <br />
                                <br/>
                                <p style="color:black"> under review </p>
                                [
                                <a href="https://arxiv.org/abs/1902.10658" style="color:#268fd6">arXiv</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/UnsupervisedAttentionMechanism" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Inspired by the adaptation phenomenon of neuronal firing, we propose an unsupervised attention mechanism (UAM) which computes the statistical regularity in the implicit space of neural networks under the Minimum Description Length (MDL) principle. Treating the neural network optimization process as a partially observable model selection problem, UAM constrained the implicit space by a normalization factor, the universal code length. We compute this universal code incrementally across neural network layers and demonstrated the flexibility to include data priors such as top-down attention and other oracle information. Empirically, our approach outperforms existing normalization methods in tackling limited, imbalanced and nonstationary input distribution in computer vision and reinforcement learning tasks. Lastly, UAM tracks dependency and critical learning stages across layers and recurrent time steps of deep networks. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/gonzalez2019enhance.png" alt="gonzalez2019enhance" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [20]&nbsp; <b class="text-primary" id="p20"> Elongated arm reach after a Pinocchio illusion in Virtual Reality &nbsp; </b>
                                <br /> Mar Gonzalez-Franco, Christopher Berger, <b><span style="color: dimgray;">Baihan Lin</span></b>, Bigna Lenggenhager, and Jaron Lanier
                                <br />
                                <br/>
                                <p style="color:black"> manuscript in preparation </p>
                                [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Perceived extension of the reach of one’s arm was observed in healthy humans after eliciting a Pinocchio Illusion—whereby a participant’s nose is illusorily elongated—in Virtual Reality. Self-administered tapping on the tip of the nose of the participants’ virtual avatar (using a hand-held controller) while the nose of the avatar slowly grew, produced a Pinocchio illusion an implicit outward drift in the participants’ arm, and the elongation of the perceived reach of their arm. We found a positive correlation between the extent of the implicit outward drift of the participants arm and the perceived reachability of distal objects. These results were found, both when the participants’ real nose was stimulated and when it was not. These findings are discussed in the context of theories on body schema and ownership, and open new avenues for hand grasp interaction with virtual objects out of arm’s-reach inside immersive setups. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2020astory.gif" alt="lin2020astory" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [19]&nbsp;<b class="text-primary" id="p19"> A Story of Two Streams: Reinforcement Learning Models from Human Behavior and Neuropsychiatry &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>, Guillermo Cecchi, Djallel Bouneffouf, Jenna Reinen, Irina Rish
                                <br />
                                <br/>
                                <p style="color:black"> AAMAS 2020 </p>
                                [
                                <a href="https://arxiv.org/abs/1906.11286" style="color:#268fd6">arXiv</a> ]&nbsp; [
                                <!-- <a href="./pdfs/NIPS_HumanQL_poster.pdf" style="color:#268fd6">poster</a> ]&nbsp; [ -->
                                <a href="./bibTex/lin2020astory.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/mentalRL" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Drawing an inspiration from behavioral studies of human decision making, we propose here a more general and flexible parametric framework for reinforcement learning that extends standard Q-learning to a two-stream model for processing positive and negative rewards, and allows to incorporate a wide range of reward-processing biases -- an important component of human decision making which can help us better understand a wide spectrum of multi-agent interactions in complex real-world socioeconomic systems, as well as various neuropsychiatric conditions associated with disruptions in normal reward processing. From the computational perspective, we observe that the proposed Split-QL model and its clinically inspired variants consistently outperform standard Q-Learning and SARSA methods, as well as recently proposed Double Q-Learning approaches, on simulated tasks with particular reward distributions, a real-world dataset capturing human decision-making in gambling tasks, and the Pac-Man game in a lifelong learning setting across different reward stationarities. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                </ul>
                <h3> 2019 </h3>
                <br />
                <ul class="fa-ul mb-0">
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2019reinforce.png" alt="lin2019reinforce" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [18]&nbsp;<b class="text-primary" id="p18"> Reinforcement Learning Models of Human Behavior: Reward Processing in Mental Disorders &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>, Guillermo Cecchi, Djallel Bouneffouf, Jenna Reinen, Irina Rish
                                <br />
                                <br/>
                                <p style="color:black"> NeurIPS 2019 Workshop on Biological & Artificial RL (BARL) </p>
                                [
                                <a href="https://arxiv.org/abs/1906.11286" style="color:#268fd6">arXiv</a> ]&nbsp; [
                                <a href="./pdfs/NIPS_HumanQL_poster.pdf" style="color:#268fd6">poster</a> ]&nbsp; [
                                <a href="./bibTex/lin2019reinforce.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/mentalRL" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Drawing an inspiration from behavioral studies of human decision making, we propose here a more general and flexible parametric framework for reinforcement learning that extends standard Q-learning to a two-stream model for processing positive and negative rewards, and allows to incorporate a wide range of reward-processing biases -- an important component of human decision making which can help us better understand a wide spectrum of multi-agent interactions in complex real-world socioeconomic systems, as well as various neuropsychiatric conditions associated with disruptions in normal reward processing. From the computational perspective, we observe that the proposed Split-QL model and its clinically inspired variants consistently outperform standard Q-Learning and SARSA methods, as well as recently proposed Double Q-Learning approaches, on simulated tasks with particular reward distributions, a real-world dataset capturing human decision-making in gambling tasks, and the Pac-Man game in a lifelong learning setting across different reward stationarities. Moreover, from the behavioral modeling perspective, our parametric framework can be viewed as a first step towards a unifying computational model capturing reward processing abnormalities across multiple mental conditions and user preferences in long-term recommendation systems. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2019what.png" alt="lin2019what" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [17]&nbsp;<b class="text-primary" id="p17"> What About Higher-Order Cellular Complexity? An Inquiry with Topological Simplicial Analysis &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>, Raul Rabadan, Nikolaus Kriegeskorte
                                <br />
                                <br/>
                                <p style="color:black"> NeurIPS 2019 Workshop on Learning Meaningful Representations of Life (LMRL) </p>
                                <!-- [<a href="" style="color:#268fd6">arXiv</a>]&nbsp; -->
                                <!-- [<a href="./bibTex/lin2019reinforce.txt" style="color:#268fd6">bibTex</a>]&nbsp; -->
                                [ arXiv to come ]&nbsp; [
                                <a href="./pdfs/NIPS_scTSA_poster.pdf" style="color:#268fd6">poster</a> ]&nbsp; [
                                <a href="./bibTex/lin2019what.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/scTSA" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> The lack of a formal link between cell-cell cohabitation and its emergent dynamics into cliques during development has hampered our understanding of how cell populations proliferate, differentiate, and compete, i.e. the cell ecology. With the advancement of single-cell RNA-sequencing (RNA-seq), we have now come closer to describing such a link by taking cell-specific transcriptional programs into account, constructing graphs of a network that reflect the similarity of gene expression, and analyzing these graphs using algebraic topology. We proposed single-cell topological simplicial analysis (scTSA). Applying this approach to single-cell gene expression profiles from local networks of cells in different developmental stages with different outcomes revealed a previously unseen topology of cellular ecology. These networks contain an abundance of cliques of single-cell profiles bound into cavities that guide the emergence of more complicated habitation forms. We visualize these ecological patterns with topological simplicial architectures of these networks, compared with the null models. Benchmarked on single-cell RNA-seq of zebrafish embryogenesis over 25 cell types and 12 time steps, our approach highlights the gastrulation as the most critical stage, consistent with consensus in developmental biology. As a nonlinear, model-independent, and unsupervised framework, our approach can also be applied to tracing multi-scale cell lineage, identifying critical stages, or creating pseudo-time series. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/andelin2019lack.png" alt="andelin2019lack" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [16]&nbsp;<b class="text-primary" id="p16"> Influence of Ocular Dominance Columns and Patchy Callosal Connections on Binocularity in Lateral Striate Cortex: Long Evans vs. Albino Rats &nbsp; </b>
                                <br /> Adrian Andelin, Zane Doyle, Robyn Laing, Joseph Turecek, <b><span style="color: dimgray;">Baihan Lin</span></b>, and Jaime Olavarria
                                <br />
                                <br/>
                                <p style="color:black"> Journal of Comparative Neurology </p>
                                [
                                <a href="./pdfs/andelin2019lack.pdf" style="color:#268fd6">pdf</a> ]&nbsp; [
                                <a href="./bibTex/andelin2019lack.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/OLab_IntanEphys" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> In albino rats, it has been reported that lateral striate cortex (V1) is highly binocular, and that input from the ipsilateral eye to this region comes through the callosum. In contrast, in Long Evans rats, this region is nearly exclusively dominated by the contralateral eye even though it is richly innervated by the callosum (Laing, Turecek, Takahata, & Olavarria, 2015). We hypothesized that the inability of callosal connections to relay ipsilateral eye input to lateral V1 in Long Evans rats is a consequence of the existence of ocular dominance columns (ODCs), and of callosal patches in register with ipsilateral ODCs in the binocular region of V1 (Laing et al., 2015). We therefore predicted that in albino rats input from both eyes intermix in the binocular region, without segregating into ODCs, and that callosal connections are not patchy. Confirming our predictions, we found that inputs from both eyes, studied with the transneuronal tracer WGA-HRP, are intermixed in the binocular zone of albinos, without segregating into ODCs. Similarly, we found that callosal connections in albino rats are not patchy but instead are distributed homogeneously throughout the callosal region in V1. We propose that these changes allow the transcallosal passage of ipsilateral eye input to lateral striate cortex, increasing its binocularity. Thus, the binocular region in V1 of albino rats includes lateral striate cortex, being therefore about 25% larger in area than the binocular region in Long Evans rats. Our findings provide insight on the role of callosal connections in generating binocular cells. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2019modeling.png" alt="lin2019modeling" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [15] &nbsp; <b class="text-primary" id="p15"> Modeling Neurological and Psychiatric Disorders with Reward Biased Reinforcement Learning Models &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>.
                                <br />
                                <br/>
                                <p style="color:black"> TIPS 2019 </p>
                                [
                                <a href="https://www.mcleanhospital.org/technology-psychiatry-summit" style="color:#268fd6">link</a> ]&nbsp; [
                                <a href="./bibTex/lin2019modeling.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/mentalRL" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Drawing an inspiration from behavioral studies of human decision making, we propose here a general parametric framework for a reinforcement learning problem, which extends the standard Q-learning approach to incorporate a two-stream framework of reward processing with biases biologically associated with several neurological and psychiatric conditions, including Parkinson's and Alzheimer's diseases, attention-deficit/hyperactivity disorder (ADHD), addiction, and chronic pain. For AI community, the development of agents that react differently to different types of rewards can enable us to understand a wide spectrum of multi-agent interactions in complex real-world socioeconomic systems. Empirically, the proposed model outperforms Q-Learning and Double Q-Learning in artificial scenarios with certain reward distributions and real-world human decision making gambling tasks. Moreover, from the behavioral modeling perspective, our parametric framework can be viewed as a first step towards a unifying computational model capturing reward processing abnormalities across multiple mental conditions and user preferences in long-term recommendation systems. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2019visualize.png" alt="lin2019visualize" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [14]&nbsp; <b class="text-primary" id="p14"> Visualizing Representational Dynamics with Multidimensional Scaling Alignment &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>, Marieke Mur, Tim Kietzmann, Nikolaus Kriegeskorte
                                <br />
                                <br/>
                                <p style="color:black"> CCN 2019 </p>
                                [
                                <a href="https://arxiv.org/abs/1906.09264" style="color:#268fd6">arXiv</a> ]&nbsp; [
                                <a href="https://ccneuro.org/2019/Papers/ViewPapers.asp?PaperNum=1424" style="color:#268fd6">link</a> ]&nbsp; [
                                <a href="./bibTex/lin2019visualize.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/rsagroup/pyrsa" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Representational similarity analysis (RSA) has been shown to be an effective framework to characterize brain-activity profiles and deep neural network activations as representational geometry by computing the pairwise distances of the response patterns as a representational dissimilarity matrix (RDM). However, how to properly analyze and visualize the representational geometry as dynamics over the time course from stimulus onset to offset is not well understood. In this work, we formulated the pipeline to understand representational dynamics with RDM movies and Procrustes-aligned Multidimensional Scaling (pMDS), and applied it to neural recording of monkey IT cortex. Our results suggest that the the multidimensional scaling alignment can genuinely capture the dynamics of the category-specific representation spaces with multiple visualization possibilities, and that object categorization may be hierarchical, multi-staged, and oscillatory (or recurrent). </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2019cliques.png" alt="lin2019cliques" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [13]&nbsp; <b class="text-primary" id="p13"> Cliques of single-cell RNA-seq profiles reveal insights into cell ecology during development and differentiation &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>
                                <br />
                                <br/>
                                <p style="color:black"> ISMB 2019 </p>
                                [
                                <a href="./pdfs/ISMB_scTSA_slides.pdf" style="color:#268fd6">slides</a> ]&nbsp; [
                                <a href="./pdfs/ISMB_scTSA_poster.pdf" style="color:#268fd6">poster</a> ]&nbsp; [
                                <a href="https://www.iscb.org/cms_addon/conferences/ismbeccb2019/posters.php?track=BioVis%20COSI&session=A" style="color:#268fd6">link</a> ]&nbsp; [
                                <a href="./bibTex/lin2019cliques.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/scTSA" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> The lack of a formal link between cell-cell cohabitation and its emergent dynamics into cliques during development has hampered our understanding of how cell populations proliferate, differentiate, and compete, i.e. the cell ecology. With the advancement of single-cell RNA-sequencing (RNA-seq), we have now come closer to describing such a link by taking cell-specific transcriptional programs into account, constructing graphs of a network that reflect the similarity of gene expression, and analyzing these graphs using algebraic topology. We proposed single-cell topological simplicial analysis (scTSA). Applying this approach to single-cell gene expression profiles from local networks of cells in different developmental stages with different outcomes revealed a previously unseen topology of cellular ecology. These networks contain an abundance of cliques of single-cell profiles bound into cavities that guide the emergence of more complicated habitation forms. We visualize these ecological patterns with topological simplicial architectures of these networks, compared with the null models. Benchmarked on single-cell RNA-seq of zebrafish embryogenesis over 25 cell types and 12 time steps, our approach highlights the gastrulation as the most critical stage, consistent with consensus in developmental biology. As a nonlinear, model-independent, and unsupervised framework, our approach can also be applied to tracing multi-scale cell lineage, identifying critical stages, or creating pseudo-time series. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/chen2019self.png" alt="chen2019self" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [12]&nbsp; <b class="text-primary" id="p12"> Self-assembling 2d arrays with de novo protein building blocks &nbsp; </b>
                                <br /> Zibo Chen, Matthew Johnson, Jiajun Chen, Matthew Bick, Scott Boyken, <b><span style="color: dimgray;">Baihan Lin</span></b>, James DeYoreo, Justin Kollman, David Baker, and Frank DiMaio
                                <br />
                                <br/>
                                <p style="color:black"> Journal of the American Chemical Society </p>
                                [
                                <a href="./pdfs/chen2019self.pdf" style="color:#268fd6">pdf</a> ]&nbsp; [
                                <a href="https://pubs.acs.org/toc/jacsat/141/22" style="color:#268fd6">JACS cover story</a> ]&nbsp; [
                                <a href="./bibTex/chen2019self.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh?&tab=repositories&q=baker" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Modular self-assembly of biomolecules in two dimensions (2D) is straightforward with DNA but has been difficult to realize with proteins, due to the lack of modular specificity similar to Watson–Crick base pairing. Here we describe a general approach to design 2D arrays using de novo designed pseudosymmetric protein building blocks. A homodimeric helical bundle was reconnected into a monomeric building block, and the surface was redesigned in Rosetta to enable self-assembly into a 2D array in the C12 layer symmetry group. Two out of ten designed arrays assembled to micrometer scale under negative stain electron microscopy, and displayed the designed lattice geometry with assembly size up to 100 nm under atomic force microscopy. The design of 2D arrays with pseudosymmetric building blocks is an important step toward the design of programmable protein self-assembly via pseudosymmetric patterning of orthogonal binding interfaces. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2019split.png" alt="lin2019split" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [11]&nbsp; <b class="text-primary" id="p11"> Split Q Learning: Reinforcement Learning with Two-Stream Rewards &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>, Djallel Bouneffouf, Guillermo Cecchi
                                <br />
                                <br/>
                                <p style="color:black"> IJCAI 2019 </p>
                                [
                                <a href="https://arxiv.org/abs/1906.12350" style="color:#268fd6">arXiv</a> ]&nbsp; [
                                <a href="./pdfs/IJCAI_SQL_slides.pdf" style="color:#268fd6">slides</a> ]&nbsp; [
                                <a href="./pdfs/IJCAI_SQL_poster.pdf" style="color:#268fd6">poster</a> ]&nbsp; [
                                <a href="./bibTex/lin2019split.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/mentalRL" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Drawing an inspiration from behavioral studies of human decision making, we propose here a general parametric framework for a reinforcement learning problem, which extends the standard Q-learning approach to incorporate a two-stream framework of reward processing with biases biologically associated with several neurological and psychiatric conditions, including Parkinson's and Alzheimer's diseases, attention-deficit/hyperactivity disorder (ADHD), addiction, and chronic pain. For AI community, the development of agents that react differently to different types of rewards can enable us to understand a wide spectrum of multi-agent interactions in complex real-world socioeconomic systems. Moreover, from the behavioral modeling perspective, our parametric framework can be viewed as a first step towards a unifying computational model capturing reward processing abnormalities across multiple mental conditions and user preferences in long-term recommendation systems. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2019neural.png" alt="lin2019neural" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [10]&nbsp; <b class="text-primary" id="p10"> Neural Networks as Model Selection with Incremental MDL Normalization &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>
                                <br />
                                <br/>
                                <p style="color:black"> IJCAI 2019 Workshop on Human Brain and Artificial Intelligence (HBAI) </p>
                                [
                                <a href="https://arxiv.org/abs/1902.10658v7" style="color:#268fd6">arXiv</a> ]&nbsp; [
                                <a href="./pdfs/IJCAI_RN_slides.pdf" style="color:#268fd6">slides</a> ]&nbsp; [
                                <a href="./pdfs/IJCAI_RN_poster.pdf" style="color:#268fd6">poster</a> ]&nbsp; [
                                <a href="http://www.ijcai-hbai.org/" style="color:#268fd6">link</a> ]&nbsp; [
                                <a href="./bibTex/lin2019neural.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/UnsupervisedAttentionMechanism" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> If we consider the neural network optimization process as a model selection problem, the implicit space can be constrained by the normalizing factor, the minimum description length of the optimal universal code. Inspired by the adaptation phenomenon of biological neuronal firing, we propose a class of reparameterization of the activation in the neural network that take into account the statistical regularity in the implicit space under the Minimum Description Length (MDL) principle. We introduce an incremental version of computing this universal code as normalized maximum likelihood and demonstrated its flexibility to include data prior such as top-down attention and other oracle information and its compatibility to be incorporated into batch normalization and layer normalization. The empirical results showed that the proposed method outperforms existing normalization methods in tackling the limited and imbalanced data from a non-stationary distribution benchmarked on computer vision and reinforcement learning tasks. As an unsupervised attention mechanism given input data, this biologically plausible normalization has the potential to deal with other complicated real-world scenarios as well as reinforcement learning setting where the rewards are sparse and non-uniform. Further research is proposed to discover these scenarios and explore the behaviors among different variants. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                </ul>
                <h3> 2018</h3>
                <br />
                <ul class="fa-ul mb-0">
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2018adgtic.png" alt="lin2018adgtic" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [9]&nbsp; <b class="text-primary" id="p9"> Adaptive Independence Tests with Geo-Topological Transformation &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>, Nikolaus Kriegeskorte
                                <br />
                                <br/>
                                <p style="color:black"> arXiv </p>
                                [
                                <a href="https://arxiv.org/abs/1810.02923" style="color:#268fd6">arXiv</a> ]&nbsp; [
                                <a href="./bibTex/lin2018adgtic.txt" style="color:#268fd6">bibTex</a> ]&nbsp;
                                <!-- [<a href="" style="color:#268fd6">code</a>]&nbsp;  -->
                                [ git to come ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Testing two potentially multivariate variables for statistical dependence on the basis finite samples is a fundamental statistical challenge. Here we explore a family of tests that adapt to the complexity of the relationship between the variables, promising robust power across scenarios. Building on the distance correlation, we introduce a family of adaptive independence criteria based on nonlinear monotonic transformations of distances. We show that these criteria, like the distance correlation and RKHS-based criteria, provide dependence indicators. We propose a class of adaptive (multi-threshold) test statistics, which form the basis for permutation tests. These tests empirically outperform some of the established tests in average and worst-case statistical sensitivity across a range of univariate and multivariate relationships and might deserve further exploration. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2018adaptive.jpg" alt="lin2018adaptive" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [8]&nbsp; <b class="text-primary" id="p8"> Adaptive Representation Selection in Contextual Bandit &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>, Guillermo Cecchi, Djallel Bouneffouf, Irina Rish
                                <br />
                                <br/>
                                <p style="color:black"> arXiv </p>
                                [
                                <a href="https://arxiv.org/abs/1802.00981" style="color:#268fd6">arXiv</a> ]&nbsp; [
                                <a href="./bibTex/lin2018adaptive.txt" style="color:#268fd6">bibTex</a> ]&nbsp;
                                <!-- [<a href="" style="color:#268fd6">code</a>]&nbsp;  -->
                                [ git to come ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> We consider an extension of the contextual bandit setting, motivated by several practical applications, where an unlabeled history of contexts can become available for pre-training before the online decision-making begins. We propose an approach for improving the performance of contextual bandit in such setting, via adaptive, dynamic representation learning, which combines offline pre-training on unlabeled history of contexts with online selection and modification of embedding functions. Our experiments on a variety of datasets and in different nonstationary environments demonstrate clear advantages of our approach over the standard contextual bandit. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2018contextual.png" alt="lin2018contextual" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [7]&nbsp; <b class="text-primary" id="p7"> Contextual Bandit with Adaptive Feature Extraction &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>, Guillermo Cecchi, Djallel Bouneffouf, Irina Rish
                                <br />
                                <br/>
                                <p style="color:black"> IEEE ICDM 2018 Workshop on Data Science and Big Data Analytics (DSBDA) </p>
                                [
                                <a href="./pdfs/lin2018contextual.pdf" style="color:#268fd6">pdf</a> ]&nbsp; [
                                <a href="./bibTex/lin2018contextual.txt" style="color:#268fd6">bibTex</a> ]&nbsp;
                                <!-- [<a href="" style="color:#268fd6">code</a>]&nbsp;  -->
                                [ git to come ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> We consider an online decision making setting known as contextual bandit problem, and propose an approach for improving contextual bandit performance by using an adap- tive feature extraction (representation learning) based on online clustering. Our approach starts with an off-line pre-training on unlabeled history of contexts (which can be exploited by our approach, but not by the standard contextual bandit), followed by an online selection and adaptation of encoders. Specifically, given an input sample (context), the proposed approach selects the most appropriate encoding function to extract a feature vector which becomes an input for a contextual bandit, and updates both the bandit and the encoding function based on the context and on the feedback (reward). Our experiments on a variety of datasets, and both in stationary and non-stationary environments of several kinds demonstrate clear advantages of the proposed adaptive representation learning over the standard contextual bandit based on "raw" input contexts. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                </ul>
                <h3> 2017</h3>
                <br />
                <ul class="fa-ul mb-0">
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/bukkittu2017parkinson.png" alt="bukkittu2017parkinson" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [6] &nbsp; <b class="text-primary" id="p6"> Parkinson’s disease digital biomarker discovery with optimized transitions and inferred markov emissions &nbsp; </b>
                                <br /> Avinash Bukkittu, <b><span style="color: dimgray;">Baihan Lin</span></b>, Trung Vu, and Itsik Pe’er
                                <br />
                                <br/>
                                <p style="color:black"> RECOMB/ISCB 2017 Conference on Regulatory & Systems Genomics </p>
                                [
                                <a href="https://arxiv.org/abs/1711.04078" style="color:#268fd6">arXiv</a> ]&nbsp; [
                                <a href="./pdfs/bukkittu2017parkinson_slides.pdf" style="color:#268fd6">slides</a> ]&nbsp; [
                                <a href="./bibTex/bukkittu2017parkinson.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/DREAM_PDBiomarker" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> We search for digital biomarkers from Parkinson's Disease by observing approximate repetitive patterns matching hypothesized step and stride periodic cycles. These observations were modeled as a cycle of hidden states with randomness allowing deviation from a canonical pattern of transitions and emissions, under the hypothesis that the averaged features of hidden states would serve to informatively characterize classes of patients/controls. We propose a Hidden Semi-Markov Model (HSMM), a latent-state model, emitting 3D-acceleration vectors. Transitions and emissions are inferred from data. We fit separate models per unique device and training label. Hidden Markov Models (HMM) force geometric distributions of the duration spent at each state before transition to a new state. Instead, our HSMM allows us to specify the distribution of state duration. This modified version is more effective because we are interested more in each state's duration than the sequence of distinct states, allowing inclusion of these durations the feature vector. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/teng2017dynamic.png" alt="teng2017dynamic" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [5] &nbsp; <b class="text-primary" id="p5"> Dynamic Forecasting of Zika Epidemics Using Google Trends &nbsp; </b>
                                <br /> Yue Teng, Dehua Bi, Guigang Xie, Yuan Jin, Yong Huang, <b><span style="color: dimgray;">Baihan Lin</span></b>, Xiaoping An, Dan Feng, and Yigang Tong
                                <br />
                                <br/>
                                <p style="color:black"> PLoS One </p>
                                [
                                <a href="./pdfs/teng2017dynamic.pdf" style="color:#268fd6">pdf</a> ]&nbsp; [
                                <a href="./bibTex/teng2017dynamic.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [ git to come ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> We developed a dynamic forecasting model for Zika virus (ZIKV), based on real-time online search data from Google Trends (GTs). It was designed to provide Zika virus disease (ZVD) surveillance and detection for Health Departments, and predictive numbers of infection cases, which would allow them sufficient time to implement interventions. In this study, we found a strong correlation between Zika-related GTs and the cumulative numbers of reported cases (confirmed, suspected and total cases; p
                                        <0.001). Then, we used the correlation data from Zika-related online search in GTs and ZIKV epidemics between 12 February and 20 October 2016 to construct an autoregressive integrated moving average (ARIMA) model (0, 1, 3) for the dynamic estimation of ZIKV outbreaks. The forecasting results indicated that the predicted data by ARIMA model, which used the online search data as the external regressor to enhance the forecasting model and assist the historical epidemic data in improving the quality of the predictions, are quite similar to the actual data during ZIKV epidemic early November 2016. Integer-valued autoregression provides a useful base predictive model for ZVD cases. This is enhanced by the incorporation of GTs data, confirming the prognostic utility of search query based surveillance. This accessible and flexible dynamic forecast model could be used in the monitoring of ZVD to provide advanced warning of future ZIKV outbreaks. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/teng2017model.png" alt="teng2017model" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [4] &nbsp; <b class="text-primary" id="p4"> Model-informed risk assessment for Zika virus outbreaks in the Asia-Pacific regions &nbsp; </b>
                                <br /> Yue Teng, Dehua Bi, Guigang Xie, Yuan Jin, Yong Huang, <b><span style="color: dimgray;">Baihan Lin</span></b>, Xiaoping An, Yigang Tong, and Dan Feng
                                <br />
                                <br/>
                                <p style="color:black"> Journal of Infection </p>
                                [
                                <a href="./pdfs/teng2017model.pdf" style="color:#268fd6">pdf</a> ]&nbsp; [
                                <a href="./bibTex/teng2017model.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Recently, Zika virus (ZIKV) has been recognized as a significant threat to global public health. The disease was present in large parts of the Americas, the Caribbean, and also the western Pacific area with southern Asia during 2015 and 2016. However, little is known about the factors affecting the transmission of ZIKV. We used Gradient Boosted Regression Tree models to investigate the effects of various potential explanatory variables on the spread of ZIKV, and used current with historical information from a range of sources to assess the risks of future ZIKV outbreaks. Our results indicated that the probability of ZIKV outbreaks increases with vapor pressure, the occurrence of Dengue virus, and population density but decreases as health expenditure, GDP, and numbers of travelers. The predictive results revealed the potential risk countries of ZIKV infection in the Asia-Pacific regions between October 2016 and January 2017. We believe that the high-risk conditions would continue in South Asia and Australia over this period. By integrating information on eco-environmental, social-economical, and ZIKV-related niche factors, this study estimated the probability for locally acquired mosquito-borne ZIKV infections in the Asia-Pacific region and improves the ability to forecast, and possibly even prevent, future outbreaks of ZIKV. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/teng2017integrative.png" alt="teng2017integrative" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [3] &nbsp; <b class="text-primary" id="p3"> An Integrative Analysis Reveals a Central Role of P53 Activation via MDM2 in Zika Virus Infection Induced Cell Death &nbsp; </b>
                                <br /> Yue Teng, Shufeng Liu, Xiaocan Guo, Shuxia Liu, Yuan Jin, Tongtong He, Dehua Bi, Pei Zhang, <b><span style="color: dimgray;">Baihan Lin</span></b>, Xiaoping An, Dan Feng, Zhiqiang Mi and Yigang Tong
                                <br />
                                <br/>
                                <p style="color:black"> Frontiers in cellular and infection microbiology </p>
                                [
                                <a href="./pdfs/teng2017integrative.pdf" style="color:#268fd6">pdf</a> ]&nbsp; [
                                <a href="./bibTex/teng2017integrative.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Zika virus (ZIKV) infection is an emerging global threat that is suspected to be associated with fetal microcephaly. However, the molecular mechanisms underlying ZIKV disease pathogenesis in humans remain elusive. Here, we investigated the human protein interaction network associated with ZIKV infection using a systemic virology approach, and reconstructed the transcriptional regulatory network to analyze the mechanisms underlying ZIKV-elicited microcephaly pathogenesis. The bioinformatics findings in this study show that P53 is the hub of the genetic regulatory network for ZIKV-related and microcephaly-associated proteins. Importantly, these results imply that the ZIKV capsid protein interacts with mouse double-minute-2 homolog (MDM2), which is involved in the P53-mediated apoptosis pathway, activating the death of infected neural cells. We also found that synthetic mimics of the ZIKV capsid protein induced cell death in vitro and in vivo. This study provides important insight into the relationship between ZIKV infection and brain diseases. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                </ul>
                <h3> 2016</h3>
                <br />
                <ul class="fa-ul mb-0">
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/lin2016odc.jpg" alt="lin2016odc" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [2] &nbsp; <b class="text-primary" id="p2"> Ocular Dominance Columns in Rat Visual Cortex: a Quantitative Model to Analyze Deprivation-Induced Cortical Plasticity &nbsp; </b>
                                <br />
                                <b><span style="color: dimgray;">Baihan Lin</span></b>, Adrian K Andelin, Jaime F Olavarria
                                <br />
                                <br/>
                                <p style="color:black"> NeuroFutures Conference 2016 </p>
                                [
                                <a href="./pdfs/NeuroFutures_ODC.pdf" style="color:#268fd6">pdf</a> ]&nbsp; [
                                <a href="https://alleninstitute.org/what-we-do/brain-science/events-training/neurofutures-2016/" style="color:#268fd6">link</a> ]&nbsp; [
                                <a href="./bibTex/lin2016odc.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a href="https://github.com/doerlbh/OLab_patch_processor_2.0" style="color:#268fd6">code</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> What is the effect of sensory deprivation on brain development? Ocular Dominance Columns (ODCs) have been extensively used to study the mechanism of cortical plasticity, despite specific functions of ODCs elusive. Using a combination of transneuronal tracing, in situ hybridization for the immediate early gene Zif268 and electrophysiological recordings, our lab recently showed that the primary visual cortex (V1) in pigmented rats has ODCs correlated with callosal inputs from the opposite hemispheres. Using similar methods, my project aims to understand the effect of monocular deprivation (MD) on the newly discovered system of ODCs in rat visual cortex. I hypothesized that the manipulation of visual input would impact the development and recovery of eye-specific circuitry. By disrupting the visual input permanently or temporarily in early postnatal life, our lab has been able to measure the deleterious effects on map development, critical periods for these effects and potential for recovery. My project used monocular eyelid suture and enucleation to study the effect of MD on the organization of ODCs in pigmented rats. By using our experience-dependent model we expect to elucidate the principles and mechanisms underlying the development of cortical modular architecture in mammals. My result showed a clear shift in Hubel-Wiesel ocular dominance scale in central segment and clear desegregation in ODC in MD rats. I also developed a Java-based software and Matlab protocol to quantify this model. As our hypothesis is supported, our novel model may shed new lights on plasticity research, which can guide future clinical studies on the treatment and prognosis of patients suffering eye defects during early development by either innate or incidental pathological causes. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                </ul>
                <h3> 2015</h3>
                <br />
                <ul class="fa-ul mb-0">
                    <li>
                        <div class="row">
                            <div class="imgcolumn">
                                <img src="./img/publications/teng2015systematic.png" alt="teng2015systematic" border="1" style="width:100%">
                            </div>
                            <div class="textcolumn">
                                [1] &nbsp; <b class="text-primary" id="p1"> Systematic Genome-wide Screening and Prediction of microRNAs in EBOV During the 2014 Ebolavirus Outbreak &nbsp; </b>
                                <br /> Yue Teng, Yuzhuo Wang, Xianglilan Zhang, Wenli Liu, Hang Fan, Hongwu Yao, <b><span style="color: dimgray;">Baihan Lin</span></b>, Ping Zhu, Wenjun Yuan, Yigang Tong and Wuchun Cao
                                <br />
                                <br/>
                                <p style="color:black"> Scientific Reports </p>
                                [
                                <a href="./pdfs/teng2015systematic.pdf" style="color:#268fd6">pdf</a> ]&nbsp; [
                                <a href="./bibTex/teng2015systematic.txt" style="color:#268fd6">bibTex</a> ]&nbsp; [
                                <a class="collapsible" style="color:#268fd6">abstract&nbsp;</a> ]
                                <div class="abstract">
                                    <p> Recently, several thousand people have been killed by the Ebolavirus disease (EVD) in West Africa, yet no current antiviral medications and treatments are available. Systematic investigation of ebolavirus whole genomes during the 2014 outbreak may shed light on the underlying mechanisms of EVD development. Here, using the genome-wide screening in ebolavirus genome sequences, we predicted four putative viral microRNA precursors (pre-miRNAs) and seven putative mature microRNAs (miRNAs). Combing bioinformatics analysis and prediction of the potential ebolavirus miRNA target genes, we suggest that two ebolavirus coding possible miRNAs may be silence and down-regulate the target genes NFKBIE and RIPK1, which are the central mediator of the pathways related with host cell defense mechanism. Additionally, the ebolavirus exploits the miRNAs to inhibit the NF-kB and TNF factors to evade the host defense mechanisms that limit replication by killing infected cells, or to conversely trigger apoptosis as a mechanism to increase virus spreading. This is the first study to use the genome-wide scanning to predict microRNAs in the 2014 outbreak EVD and then to apply systematic bioinformatics to analyze their target genes. We revealed a potential mechanism of miRNAs in ebolavirus infection and possible therapeutic targets for Ebola viral infection treatment. </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <br />
                </ul>
            </div>
        </section>
        <section class="resume-section p-3 p-lg-5 d-flex flex-column" id="life">
            <div class="my-auto">
                <h2 class="mb-5">Life</h2>
                <b class="text-primary"> I am always curious about the world and I like exploring new things. &nbsp; </b>
                <p class="mb-5 text-primary">
                    <br>Although New York is not like Seattle with all the Nature, I try to maintain my hobbies in skateboarding and longbaording (well, kayaking is obviously not gonna happen here easily). Beyond my footsteps, I enjoy traveling to different places (so far only 3 continents reached) - I am planning a roadtrip across Northern Europe some time next year. </p>
                <b class="text-primary"> Despite doing mostly theoretical research in recent days, I love making tangible things: &nbsp; </b>
                <p class="mb-5 text-primary">
                    <br> woodworking, DIY, electronics and interior design. A recent woodwork project of mine is a <a href="https://www.instagram.com/p/BnnMfQcDjWI/?utm_source=ig_web_button_share_sheet"><u>modern sofa</u></a> (<a href="https://www.instagram.com/p/Bno19-ZjVSo/?utm_source=ig_web_button_share_sheet"><u>process</u></a>). As another project under development, I am creating tangible representations of high-dimensional interaction data (more details yet to come). </p>
                <b class="text-primary"> Music! &nbsp; </b>
                <p class="mb-5 text-primary">
                    <br> Other than these life bits, I play flute and am currently learning Jazz and guitar, well, at least while my neighbors and <a href="https://www.instagram.com/neptunesnaptime/"><u>my cats</u></a> can still bear it. ;) </p>
                <p class="mb-5 text-primary">
                    <br> More bits of life can be found <a href="bits"><u><b>here</b></u></a></p>
            </div>
        </section>
        </div>

        <script src="js/jquery.min.js"></script>
        <script src="js/bootstrap.bundle.js"></script>
        <script src="js/jquery.easing.js"></script>
        <script src="js/resume.js"></script>
</body>
<!-- 
        <div class="flier2"><img src="img/random/rock.png" alt="" style="width:500px;height:300px;"></div>

        <div class="flier"><img src="img/random/doge.png" alt="" style="width:300px;height:400px;"></div>
         -->
<footer style="align:center;">&copy; Copyright 2016
    <script>
        new Date().getFullYear() > 2016 && document.write("-" + new Date().getFullYear());
    </script> by Baihan Lin</footer>

</html>